tools/Nethermind.Tools.Kute/MetricsConsumer/PrettyReportMetricsConsumer.cs

```diff
@@ -1,35 +1,20 @@
 // SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
 // SPDX-License-Identifier: LGPL-3.0-only
 
+using App.Metrics.Formatters.Ascii;
+
 namespace Nethermind.Tools.Kute.MetricsConsumer;
 
 class PrettyReportMetricsConsumer : IMetricsConsumer
 {
-    public void ConsumeMetrics(Metrics metrics)
+    public async Task ConsumeMetrics(Metrics metrics)
     {
-        Console.WriteLine($"""
-        Total Running Time: {metrics.TotalRunningTime.TotalMilliseconds} ms
-        Results:
-          Messages .. {metrics.Messages}
-            Failed .. {metrics.Failed}
-            Succeeded
-              Responses .. {metrics.Responses}
-              Requests
-                Batches .. {metrics.Batches.Count} in {metrics.Batches.RunningTime.TotalMilliseconds} ms
-                Singles
-                  Ignored .. {metrics.IgnoredRequests}
-                  Processed
-        """);
+        var snapshot = metrics.Snapshot;
+        var formatter = new MetricsTextOutputFormatter();
```

Comment:
same here.
Actually, metrics consumer seems redundant as it is basically formatter + outputting to console which could be done in a simple extension method on formatter.

Reply:
Indeed, it could be done that way but I decided to wrap it in our class in case we want to add some extra configuration to the formatters. The current output is a bit verbose in my opinion (both in JSON and Report formats) and it's a lot easier to modify/try stuff out this way.

Reply:
Code has been refactored: now we have a single `ConsoleMetricsConsumer` which deals with getting a `snapshot` and then writing to console. This consumer takes a formatter (JSON or Report) which is configured during initialization.